config/initializers/assets.rb

```diff
@@ -3,10 +3,3 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
-# Add additional assets to the asset load path
-# Rails.application.config.assets.paths << Emoji.images_path
-
-# Precompile additional assets.
-# application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-# Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w( email.css emoji/*.png admin.js )
```

Comment:
Did we want to remove `email.css` and `admin.js` too? If not, I think we'll need to include them in the `manifest.js` file.

Reply:
Nevermind, these get compiled into `public/assets`. It may only matter once we actually have a `manifest.js`?

Reply:
They're already in the proper directories so I thought that was redundant. `./app/assets/stylesheets/email.scss` and `./app/assets/javascripts/admin.js`